scripts/run_proofs.sh

```diff
@@ -10,7 +10,7 @@ git fetch origin
 error=0
 for new_proof in $(git diff origin/develop --name-only test/formal/)
 do
-	if [ -e "$new_proof" ]; then
+	if [ -e "$new_proof" ] && [[ "$new_proof" != *"README.md" ]]; then
```

Comment:
Why not?
```suggestion
	if [ -e "$new_proof" ] && [[ "$new_proof" == *.py ]]; then
```

Or are those files not saved as Python scripts?

Reply:
> Or weren't those files saved as python scripts?

Yes, they are: https://github.com/argotorg/solidity/tree/develop/test/formal

Reply:
it'll try to run the readme as python script otherwise ;)

Reply:
There's also `util.py`, which is not a test script, just utilities. I guess running it does not break anything in its current form, but perhaps we should not be assuming we can run every script there.

Reply:
oh, missed that one. Maybe we should only keep scripts in there and move `utils.py` to a better place? Or move the tests to a subdirectory? What do you think?

Reply:
I think a subdirectory would be reasonable.

Though, to be clear, it's not like it's a big issue (which is why I even did not mark it unresolved). Probably better to exclude too little than too much.